SYCL/Plugin/level_zero_interop_memcpy.cpp

```diff
@@ -0,0 +1,28 @@
+// REQUIRES: level_zero, level_zero_dev_kit
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %level_zero_options %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+#include <CL/sycl.hpp>
+#include <CL/sycl/backend/level_zero.hpp>
+#include <level_zero/ze_api.h>
+
+int main() {
+  using namespace sycl;
+
+  queue Q{gpu_selector{}};
+
+  auto nativeQ = sycl::get_native<sycl::backend::ext_oneapi_level_zero>(Q);
+
+  queue Q2 = sycl::ext::oneapi::level_zero::make<sycl::queue>(
+      Q.get_context(), nativeQ, sycl::ext::oneapi::level_zero::ownership::keep);
+
+  // Command submission works fine
+  Q2.submit([&](handler &cgh) { cgh.single_task<class K>([]() {}); });
+
+  // Copy submissions crashes with discrete GPUs.
+  int hostMem;
+  // int *hostMem = (int*)malloc_host(sizeof(int), Q2);
```

Comment:
I am sorry, I somehow missed this PR.

I am trying to understand what we are trying to check here. Is it expected to fail if hostMem is allocated using malloc_host?

Reply:
I think it is only trying to check that "memcpy" at line 27 works OK with interop-queue. I don't recall why "malloc_host" was causing problems. Will try again.